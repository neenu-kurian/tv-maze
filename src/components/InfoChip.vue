<template>
  <div class="inline-flex gap-2 items-center bg-gray-100 rounded-full px-3 py-1 text-sm">
    <span>{{ label }}</span>
    <span class="font-bold"
      ><slot>{{ value }}</slot></span
    >
  </div>
</template>

<script setup lang="ts">
import type { InfoChipProps } from '@/types/index.ts';

defineProps<InfoChipProps>();
</script>
