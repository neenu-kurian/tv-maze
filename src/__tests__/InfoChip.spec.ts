import { render } from '@testing-library/vue';
import InfoChip from '@/components/InfoChip.vue';
import { describe, it, expect } from 'vitest';

describe('InfoChip.vue', () => {
  it('matches snapshot', () => {
    const { container } = render(InfoChip, {
      props: {
        label: 'Duration',
        value: '60m'
      },
    });
    expect(container).toMatchSnapshot();
  });
});
