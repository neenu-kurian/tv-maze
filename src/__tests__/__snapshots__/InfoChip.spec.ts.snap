// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomeView.vue > matches snapshot 1`] = `
<div>
  <div
    class="inline-flex gap-2 items-center bg-gray-100 rounded-full px-3 py-1 text-sm"
  >
    <span>
      Duration
    </span>
    <span
      class="font-bold"
    >
      
      60m
      
    </span>
  </div>
</div>
`;
